mobx_codegen/pubspec.yaml

```diff
@@ -16,10 +16,15 @@ dependencies:
   path: ^1.6.2
   source_gen: ^0.9.4
 
+dependency_overrides:
+  mobx:
+    path: ../mobx
+
```

Comment:
We should remove this

Reply:
yes, this one was just to let the tests run correctly during dev

Reply:
> yes, this one was just to let the tests run correctly during dev

Agree. We can also publish to `.alpha` versions for testing purposes

Reply:
How to proceed with publishing .alpha version ?